element/index.js

```diff
@@ -65,3 +66,20 @@ export function renderToString( element ) {
 
 	return renderToStaticMarkup( element );
 }
+
+export function concatValues( value1, value2 ) {
```

Comment:
`concatChildren` ?

> If we are to drop the element React abstraction, what's to happen to these utility functions?

Yep, we should consider moving those to a utils inside the `editor` folder may be.